client/src/components/Tables/PitchDoc/index.tsx

```diff
@@ -0,0 +1,190 @@
+import React, { FC, ReactElement, useEffect, useState } from 'react';
+import { Table } from 'semantic-ui-react';
+import { IPitch, IUser } from 'ssw-common';
+
+import { FieldTag, TableTool } from '../..';
+import { useAuth, useInterests, useTeams } from '../../../contexts';
+
+import './styles.scss';
+
+interface PitchTableProps {
+  pitches: IPitch[];
+}
+
+interface PitchHeaderProps {
+  pitches: IPitch[];
+  sortedPitches: IPitch[];
+  setSortedPitches: (sortedPitches: IPitch[]) => void;
+}
+
+interface PitchBodyProps {
+  sortedPitches: IPitch[];
+}
+
+interface PitchRowProps {
+  pitch: IPitch;
+}
+
+const titleSort = (a: IPitch, b: IPitch): number =>
+  a.title.localeCompare(b.title);
+
+const descriptionSort = (a: IPitch, b: IPitch): number =>
+  a.description.localeCompare(b.description);
+
+interface ColumnEnumValue {
+  title: string;
+  sort: (a: IPitch, b: IPitch) => number;
+}
+
+const columnsEnum: { [key: string]: ColumnEnumValue } = {
+  TITLE: {
+    title: 'TITLE',
+    sort: titleSort,
+  },
+  DESCRIPTION: {
+    title: 'DESCRIPTION',
+    sort: descriptionSort,
+  },
+};
+
+const PitchHeader: FC<PitchHeaderProps> = ({
+  pitches,
+  sortedPitches,
+  setSortedPitches,
+}): ReactElement => {
+  const [column, setColumn] = useState<ColumnEnumValue>();
+  const [direction, setDirection] = useState<'ascending' | 'descending'>();
+
+  const handleSort = (newColumn: ColumnEnumValue): void => {
+    if (column?.title === newColumn.title) {
+      if (direction === 'ascending') {
+        setDirection('descending');
+        setSortedPitches(sortedPitches.slice().reverse());
+      } else if (direction === 'descending') {
+        setSortedPitches(pitches);
+        setDirection(undefined);
+        setColumn(undefined);
+      } else {
+        setDirection('ascending');
+        setSortedPitches(sortedPitches.slice().reverse());
+      }
+    } else {
+      setColumn(newColumn);
+      setDirection('ascending');
+
+      const copy = [...sortedPitches];
+      copy.sort(newColumn.sort);
+      setSortedPitches(copy);
+    }
+  };
+
+  useEffect(() => {
+    setSortedPitches(pitches);
+  }, [pitches, setSortedPitches]);
+
+  return (
+    <Table.Row>
+      <Table.HeaderCell
+        onClick={() => handleSort(columnsEnum.TITLE)}
+        sorted={column === columnsEnum.TITLE ? direction : undefined}
+      >
+        Title
+      </Table.HeaderCell>
+      <Table.HeaderCell
+        onClick={() => handleSort(columnsEnum.DESCRIPTION)}
```

Comment:
I don't think we need to be able to sort by description. @bryange3 @sahithimuthyala double check me on that